pages/linux/unshare.md

```diff
@@ -0,0 +1,12 @@
+# unshare
+
+> Execute a command in new user-defined namespaces.
+> More information: <https://www.kernel.org/doc/html/latest/userspace-api/unshare.html>.
+
+- Execute a command without sharing access to connected networks:
+
+`unshare --net {{command}}`
```

Comment:
I think this clarification is better
```suggestion
`unshare --net {{command}} {{command_arguments}}`
```

Reply:
I have mixed feelings towards this and favor what we had before.

If anyone else cares to chime in, maybe they can break the tie?

Reply:
I agree with axel.